filebeat/module/mysql/_meta/kibana/7/dashboard/Filebeat-mysql.json

```diff
@@ -28,7 +28,7 @@
                             "id": "1", 
                             "params": {
                                 "customLabel": "Query time", 
-                                "field": "mysql.slowlog.query_time.sec"
+                                "field": "event.duration"
```

Comment:
This one changes the unit, so it's not a 1-1 mapping. We should follow up to see if the dashboard still looks as expected.